### PR TITLE
merge docker and service repositories

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,38 @@
-language: java
+---
+  sudo: required
 
-script:
-  - mvn clean verify jacoco:report coveralls:report
\ No newline at end of file
+  env:
+    global:
+      - DOCKER_COMPOSE_VERSION=1.21.1
```
Comment (zmoore-usgs) [inline .travis.yml]: I think this needs to be `1.22.0` for compose version `3.7` to work

Comment (kmschoep-usgs) [reply .travis.yml]: whoops!

File: README.md [hunk 2]
```diff
@@ -26,4 +26,15 @@ default the project has both profiles. For local development, it may be convenie
...
 
-* MLR_NOTIFICATION_SERVICE_PWD - The default user account password
\ No newline at end of file
+
+## Running the Application
+
+This application can be run locally using the docker container built during the build process or by directly building and running the application JAR. The included `docker-compose` file has 3 profiles to choose from when running the application locally:
+
+1. mlr-notification-service: This is the default profile which runs the application as it would be in our cloud environment. This is not recommended for local development as it makes configuring connections to other services running locally on your machine more difficult.
+2. mlr-notification-service-local-dev: This is the profile which runs the application as it would be in the aqcu-local-dev project, and is configured to make it easy to replace the aqcu-gateway instance in the local-dev project with this instance. It is run the same as the `mlr-notification-service` profile, except it uses the docker host network driver.
```
Comment (mhines-usgs) [inline README.md]: some typos here about aqcu-gateway

Comment (kmschoep-usgs) [reply README.md]: darn it, thought I caught them all...

Comment (mhines-usgs) [reply README.md]: yeah, zack and i had fun with these last week :whack-a-mole: 😆

**Comment by mhines-usgs** [comment]
Because I'm not sure, what port is used? seems like the ports listed are all over the place 8080, 6025, 443, is that fine?

**Comment by zmoore-usgs** [comment]
Yeah, looks like we need some standardization there.

The Dockerfile defines `8444`, compose file opens `443` and the config.env file defines `6025`. I'd probably standardize those all to the value used in `mlr-local-dev`, which appears to be 6026:

https://github.com/USGS-CIDA/mlr-local-dev/blob/master/docker-compose.yml#L169

**Comment by kmschoep-usgs** [comment]
yep, just looking that up now...

**Pull request merged**